dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/sms/hibernate/SMSCode.hbm.xml

```diff
@@ -24,7 +24,8 @@
 			class="org.hisp.dhis.trackedentity.TrackedEntityAttribute" column="trackedentityattributeid"
 			foreign-key="fk_trackedentityattribute_trackedentityattributeid" />
 
-		<property name="optionId" type="integer" />
+		<many-to-one name="optionId" class="org.hisp.dhis.category.CategoryOptionCombo"
+					 column="optionid" foreign-key="fk_categoryoptioncombo_categoryoptioncomboid" />
```

Comment:
Where does this foreign key `fk_categoryoptioncombo_categoryoptioncomboid` come from?
Do we need to create it in the flyway script?